c7n/resources/account.py

```diff
@@ -1603,3 +1603,76 @@ class AccountCatalogEncryptionFilter(GlueCatalogEncryptionEnabled):
               SseAwsKmsKeyId: alias/aws/glue
 
     """
+
+
+@filters.register('emr-block-public-access-configuration')
+class GetAccountBlockPublicAccessConfiguration(Filter):
+    """Check for EMR block public access configuration on an account
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: get-emr-block-public-access-configuration
+                resource: account
+                filters:
+                  - type: emr-block-public-access-configuration
+    """
+
+    schema = type_schema('emr-block-public-access-configuration',)
+    permissions = ("elasticmapreduce:GetBlockPublicAccessConfiguration",)
+
+    def process(self, resources, event=None):
+        client = local_session(self.manager.session_factory).client('emr')
+
+        response = client.get_block_public_access_configuration()
+        response.pop('ResponseMetadata')
+        response = [response]
+
+        return response
```

Comment:
this needs some work, for filter actually filtering on public block, and return the right resource / accounts.